### PR TITLE
Adding a Contributing.rst

### PR DESCRIPTION
**Pull request by muaazsaleem**
Adding a Contributing.rst to document a few things first time contributors might find relevant.

### CODE REVIEW

**Comment by szuecs** [comment]
:-1: please use markdown

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+We only support rolling out changes to our internal clusters so far. Our
+Kubernetes clusters are divided into multiple channels. Each channel has
+a corresponding branch. For example, the ["alpha"
+branch](https://github.com/zalando-incubator/kubernetes-on-aws/tree/alpha)
+represents the state of clusters in the "alpha" channel. See [Cluster
+Provisioning](https://kubernetes-on-aws.readthedocs.io/en/latest/admin-guide/kubernetes-in-production.html#cluster-provisioning) to learn more. To roll out changes to a particular
+channel, create a PR to it's corresponding branch e.g
+[dev-to-alpha](https://github.com/zalando-incubator/kubernetes-on-aws/pull/1130).
+You may use the [prepare.sh](https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/prepare-pr.sh) script to create a PR.
```
Comment (szuecs) [inline CONTRIBUTING.md]: prepare.sh -> prepare-pr.sh

**Comment by szuecs** [comment]
:+1:

**Comment by muaazsaleem** [comment]
👍

**Comment by szuecs** [comment]
:+1:

**Comment by muaazsaleem** [comment]
👍

**Comment by muaazsaleem** [comment]
👍

**Pull request merged**